src/panels/logbook/ha-logbook.ts

```diff
@@ -188,110 +209,131 @@ class HaLogbook extends LitElement {
-      }
-
-      .time {
-        display: flex;
-        justify-content: center;
-        flex-direction: column;
-        width: 75px;
-        flex-shrink: 0;
-        font-size: 12px;
-        color: var(--secondary-text-color);
-      }
-
-      .date {
-        margin: 8px 0;
-        padding: 0 16px;
-      }
-
-      .narrow .date {
-        padding: 0 8px;
-      }
-
-      .rtl .date {
-        direction: rtl;
-      }
-
-      .icon-message {
-        display: flex;
-        align-items: center;
-      }
-
-      .no-entries {
-        text-align: center;
-        color: var(--secondary-text-color);
-      }
-
-      ha-icon {
-        margin: 0 8px 0 16px;
-        flex-shrink: 0;
-        color: var(--primary-text-color);
-      }
-
-      .message {
-        color: var(--primary-text-color);
-      }
-
-      .no-name .message:first-letter {
-        text-transform: capitalize;
-      }
-
-      a {
-        color: var(--primary-color);
-      }
-
-      .uni-virtualizer-host {
-        display: block;
-        position: relative;
-        contain: strict;
-        height: 100%;
-        overflow: auto;
-      }
-
-      .uni-virtualizer-host > * {
-        box-sizing: border-box;
-      }
-
-      .narrow .entry {
-        flex-direction: column;
-        line-height: 1.5;
-        padding: 8px 0;
-      }
-
-      .narrow .icon-message ha-icon {
-        margin-left: 0;
-      }
-    `;
+  static get styles(): CSSResultArray {
+    return [
+      haStyle,
+      css`
+        :host {
+          display: block;
+          height: 100%;
+        }
+
+        .rtl {
+          direction: ltr;
+        }
+
+        .entry-container {
+          width: 100%;
+        }
+
+        .entry {
+          display: flex;
+          width: 100%;
+          line-height: 2em;
+          padding: 8px 16px;
+          box-sizing: border-box;
+          border-top: 1px solid
+            var(--mdc-dialog-scroll-divider-color, rgba(0, 0, 0, 0.12));
+        }
+
+        .entry.no-entity,
+        .no-name .entry {
+          cursor: default;
+        }
+
+        .entry:hover {
+          background-color: rgba(var(--rgb-primary-text-color), 0.04);
+        }
+
+        .narrow:not(.no-icon) .time {
+          margin-left: 32px;
+        }
+
+        .message-relative_time {
+          display: flex;
+          flex-direction: column;
+        }
+
+        .secondary {
+          font-size: 12px;
+          line-height: 1.7;
+        }
+
+        .date {
+          margin: 8px 0;
+          padding: 0 16px;
+        }
+
+        .narrow .date {
+          padding: 0 8px;
+        }
+
+        .rtl .date {
+          direction: rtl;
+        }
+
+        .icon-message {
+          display: flex;
+          align-items: center;
+        }
+
+        .no-entries {
+          text-align: center;
+          color: var(--secondary-text-color);
+        }
+
+        ha-icon {
+          margin: 4px 8px 0 0;
```

Comment:
```suggestion
          margin-right: 16px;
```

Reply:
```suggestion
          margin-right: 8px;
```

Why do you think they icon needs to be so far from the text?

Reply:
equal from both sides and material guidelines <https://material.io/components/lists#specs>